services/src/main/java/org/keycloak/protocol/oidc/OIDCWellKnownProvider.java

```diff
@@ -160,6 +160,9 @@ public Object getConfig() {
         config.setBackchannelLogoutSupported(true);
         config.setBackchannelLogoutSessionSupported(true);
 
+        config.setGrantIdSupported(true);
```

Comment:
According to https://openid.net/specs/fapi-grant-management-01.html#section-5.1 , it might be "none", "optional" or "always".
As mentioned earlier, "optional" might be appropriate.